modules/client/test/integration/multisig-client/methods.test.ts

```diff
@@ -60,6 +63,21 @@ describe("Client Multisig", () => {
         contextParamsLocalChain.web3Providers as any,
       );
     }
+    LIVE_CONTRACTS.goerli.daoFactory = deployment.daoFactory.address;
```

Comment:
Why these ad-hoc overrides?

Reply:
The plugin installation encoders uses the active contracts values to connect to contracts